src/client/android/app/src/main/java/com/yoda/MainApplication.java

```diff
@@ -1,19 +1,28 @@
 package com.yoda;
 
 import android.app.Application;
-import android.util.Log;
 
+import com.facebook.CallbackManager;
+import com.facebook.FacebookSdk;
+import com.facebook.appevents.AppEventsLogger;
 import com.facebook.react.ReactApplication;
-import com.facebook.react.ReactInstanceManager;
 import com.facebook.react.ReactNativeHost;
 import com.facebook.react.ReactPackage;
 import com.facebook.react.shell.MainReactPackage;
+import com.facebook.reactnative.androidsdk.FBSDKPackage;
 
 import java.util.Arrays;
 import java.util.List;
 
 public class MainApplication extends Application implements ReactApplication {
 
+  // implement CallbackManager for facebook SDK
```

Comment:
Capitalize. Period at the end. 

"Implement CallbackManager for Facebook SDK."